content/docs/command-reference/plots/index.md

```diff
@@ -1,57 +1,56 @@
 # plots
 
-A set of commands to visualize and compare _plot metrics_:
+A set of commands to visualize and compare _plot data_:
 [show](/doc/command-reference/plots/show),
-[diff](/doc/command-reference/plots/diff), and
-[modify](/doc/command-reference/plots/modify).
+[diff](/doc/command-reference/plots/diff),
+[modify](/doc/command-reference/plots/modify), and
+[templates](/doc/command-reference/plots/templates).
 
 ## Synopsis
 
 ```usage
-usage: dvc plots [-h] [-q | -v] {show,diff,modify} ...
+usage: dvc plots [-h] [-q | -v] {show,diff,modify,templates} ...
 
 positional arguments:
-  COMMAND
-    show         Generate plot from a metrics file.
-    diff         Plot differences in metrics between commits.
-    modify       Modify display properties of data-series plots (has no effect on image-type plots).
+  {show,diff,modify,templates}
+                        Use `dvc plots CMD --help` to display command-specific help.
+    show                Generate plots from target files or plots definitions from `dvc.yaml` file.
+    diff                Show multiple versions of plot data by plotting them in a single image.
+    modify              Modify display properties of data-series plot outputs (has no effect on image-type plots).
+    templates           Write built-in plots templates to a directory (.dvc/plots by default).
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -q, --quiet           Be quiet.
+  -v, --verbose         Be verbose.
 ```
 
-## Types of metrics
```

Comment:
do we want to move it somewhere? (not sure if we have a good place)

Reply:
This is in https://dvc.org/doc/command-reference/metrics#types-of-metrics now. At least link from `plots` ref(s), yeah.

Reply:
As I mentioned in the [issue description](https://github.com/iterative/dvc.org/pull/3691#issue-1283821901) I don't think we should be trying to explain plots in terms of metrics types. `dvc plots` is a visualization that can be created from any type of viable data. Yes, we have a convenient output type that lets DVC pick up plots without defining them but I believe saying that its special type of metrics makes users think about it in a particular way that is unnecessary.

Reply:
Agreed, and as I mentioned I'm not sure we have a good place for this.

Reply:
@jorgeorpinel let's drop it in the second place as well?